lib/src/components/NxTag/NxTag.scss

```diff
@@ -37,22 +38,28 @@
   background-color: var(--nx-selectable-color-light);
 
   &:hover {
-    border-color: var(--nx-color-interactive-border-hover);
+    border-color: var(--nx-tag-unselected-hover-border);
+    background-color: var(--nx-tag-unselected-hover-background);
   }
 
   &:focus-within {
     &::before {
       border: 2px solid var(--nx-color-interactive-border-focus);
       border-radius: 20px;
       content: '';
-      inset: calc(0px - (var(--nx-spacing-base) + 1px));
+      inset: calc(0px - (var(--nx-spacing-base) + var(--nx-border-width)));
```

Comment:
I'm not sure this calculation makes sense.  The variables being used here don't really seem related to the value we're trying to compute, and I'm pretty sure it ends up with the wrong value anyway in light mode.  As far as I can tell it should be 4px, but is 6px in dark mode and 5px in light mode.  I believe the correct variables to use for this calculation would be `--nx-focus-outline-width` and `--nx-focus-outline-inset` (soon to be renamed)

Reply:
> I'm not sure this calculation makes sense. The variables being used here don't really seem related to the value we're trying to compute, and I'm pretty sure it ends up with the wrong value anyway. As far as I can tell it should be 4px, but is 6px in dark mode and 5px in light mode. I believe the correct variables to use for this calculation would be `--nx-focus-outline-width` and `--nx-focus-outline-inset` (soon to be renamed)

In dark-mode, setting the inset to 4px would bring the height of the focus-outline to be 36px. The mock shows it to be 40px (inset of 6px would be correct). 

In light mode, setting the inset to 4px would bring the height of the focus-outline to be 38px. The mock shows it to be 40px (inset of 5px would be correct).

Reply:
> > In dark-mode, setting the inset to 4px would bring the height of the focus-outline to be 36px. The mock shows it to be 40px (inset of 6px would be correct).
> 
> How do you figure? `inset` is a shorthand for `top`, `bottom`, `left`, and `right`. So `inset: 4px` means `top: 4px; bottom: 4px;` (and left and right). The `.nx-tag` is 32px tall. The focus border should be 40px tall. 40 - 32 = 8. 8 / 2 = 4. So I believe `top: 4px; bottom: 4px` is correct.

I could be wrong, but I think what's happening is that currently, the 32px height of the`.nx-tag` is due to 1px border on the top and bottom. An inset of 0px does not account for the 1px border, and looks like this:
<img width="313" alt="image" src="https://user-images.githubusercontent.com/7110862/235179545-434e1bed-c6d2-4d4c-9339-710323a3a2cc.png">

In order to correctly offset the 1px border, the inset needs to have an additional 1px, hence 4px+1px = 5px in light mode, and 4px+2px = 6px in dark mode (since dark mode has a 2px border instead of one).

Reply:
> > An inset of 0px does not account for the 1px border, and looks like this:
> 
> Ah that's what I was forgetting that `inset` (and `top` etc) count from the parent's padding box rather than border box. In that case your numbers are correct but your use of `var(--nx-spacing-base)` still seems not ideal. I think that should be replaced by `var(--nx-focus-outline-width) + var(--nx-focus-outline-inset)`.

Changed here: b275f34f3c3528b3ab3b613520977aebc9bd4c38